### ISSUE TITLE
How to choose latent variable when computing metrics?

### ISSUE THREAD

**Issue by hukkelas** [lyndonzheng/Pluralistic-Inpainting#49]
Hi,

I'm looking into reproducing your results using the pre-trained models on CelebA-HQ and places2.

More specifically, I would like to compute the PSNR, SSIM, L1, L2, LPIPS, and FID scores on these dataset. 
I'm able to load and run the models as expected; however, to get reproducible results I would like to know what is the optimal way to select the random latent variable.

For example, I've tried to set the vector to all 0's, but this generates bad results on the places2 dataset (seemed to work fine on CelebA-HQ).

Currently I'm setting the manual seed to torch to 0 before you generate the latent variable (which I believe is done [here](https://github.com/lyndonzheng/Pluralistic-Inpainting/blob/1ca1855615fed8b686ca218c6494f455860f9996/model/pluralistic_model.py#L93)).

By using the following input image:
https://imgur.com/a/DggGAmO
By setting the latent variable to all 0, I got this result:
https://imgur.com/a/J11D4Ip
By setting the manual seed prior to the sampling I got this:
https://imgur.com/a/ZcWr32S

**Comment by lyndonzheng** [comment]
For the quantitative comparison, we directly random generate 50 samples for a given masked image. Then, we select the closest to the original image as our final results to evaluate the PSNR, SSIM and L1.

**Comment by hukkelas** [comment]
Thanks for the answer!
